ios/FluentUI/Command Bar/CommandBarButton.swift

```diff
@@ -100,15 +100,20 @@ class CommandBarButton: UIButton {
             if isSelected {
                 backgroundColor = selectedBackgroundColor
             } else if isHighlighted {
-                backgroundColor = CommandBarButton.highlightedBackgroundColor
+                backgroundColor = ColorConstants.highlightedBackgroundColor
             } else {
-                backgroundColor = CommandBarButton.normalBackgroundColor
+                backgroundColor = ColorConstants.normalBackgroundColor
             }
         }
     }
 
-    private static let contentEdgeInsets = UIEdgeInsets(top: 8.0, left: 10.0, bottom: 8.0, right: 10.0)
-    private static let normalTintColor: UIColor = Colors.textPrimary
-    private static let normalBackgroundColor = UIColor(light: Colors.gray50, dark: Colors.gray600)
-    private static let highlightedBackgroundColor = UIColor(light: Colors.gray100, dark: Colors.gray900)
+    private struct LayoutConstants {
+        static let contentEdgeInsets = UIEdgeInsets(top: 8.0, left: 10.0, bottom: 8.0, right: 10.0)
```

Comment:
nit: can you make this indentation consistent with the other constants in the CommandBarCommandGroupsView.swift file?